docs/madr/1-intial.md

```diff
@@ -0,0 +1,155 @@
+# An event driven ANT registration system in AO to optimize ANT listings by access role
+
+- Status: accepted
+- Deciders: [Dylan], [Ariel], [Phil]
+- Date: 2024-07-22
+- Authors: [Atticus], [Dylan], [Ariel], [Phil]
+
+## Context and Problem Statement
+
+Listing all ANTs registered to ArNS names and owned by a specific owner or
+owners becomes more computationally expensive as more ArNS names are registered.
+Additionally, its computationally infeasible to find ANTs not registered to an
```

Comment:
```suggestion
Additionally, its computationally expensive and non-deterministic to find ANTs not registered to an
```

Reply:
non-deterministic bc we may not find the same results every time, using known tools like Graphql and AO